packages/calling-component-bindings/package.json

```diff
@@ -36,12 +36,12 @@
     "reselect": "~4.0.0"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.6.1-beta.1 || 1.4.4",
```

Comment:
Please separate out the calling SDK version bump into its own PR from the PR that introduces the `TeamsCall` related hierarchy?

The `TeamsCall` introduction is a sizable independenty change and I'd like to avoid other routine changes needed for the headless SDK bump from getting mixed into that change.

Reply:
Still waiting for this PR to be split for a proper review.

Please re-assign when ready.